modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/SkuSubscriptionConfigurationResourceTest.java

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.commerce.admin.catalog.resource.v1_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.commerce.product.model.CPInstance;
+import com.liferay.commerce.product.service.CPInstanceLocalServiceUtil;
+import com.liferay.commerce.product.test.util.CPTestUtil;
+import com.liferay.headless.commerce.admin.catalog.client.dto.v1_0.SkuSubscriptionConfiguration;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.UserTestUtil;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Zoltán Takács
+ * @author Crescenzo Rega
+ */
+@RunWith(Arquillian.class)
+public class SkuSubscriptionConfigurationResourceTest
+	extends BaseSkuSubscriptionConfigurationResourceTestCase {
+
+	@Before
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_user = UserTestUtil.addUser(testCompany);
+	}
+
+	@Override
+	@Test
+	public void testGetSkuByExternalReferenceCodeSubscriptionConfiguration()
```

Comment:
why do you need to override test methods?

Reply:
because an assertion to false is generated in the **BaseSkuSubscriptionConfigurationResourceTestCase** class

```
@Test
public void testPatchSkuIdSubscriptionConfiguration() throws Exception {
Assert.assertTrue(false);
}
```